x-pack/plugins/siem/public/resolver/view/process_event_dot.tsx

```diff
@@ -251,8 +256,33 @@ const ProcessEventDotComponents = React.memo(
      * position to accomodate for that. This aligns the logical center of the process node
      * with the correct position on the map.
      */
-    const processNodeViewXOffset = -0.172413 * logicalProcessNodeViewWidth * magFactorX;
-    const processNodeViewYOffset = -0.73684 * logicalProcessNodeViewHeight * magFactorX;
+
+    const isZoomedOut = magFactorX < 0.8;
```

Comment:
The `scalingFactor` is from `0` to `1` and represents, semantically (kinda) how zoomed in things are. The range slider that the user can use to control zoom is mapped to it. Depending on what you're doing here, it might be worth using that instead.

If you want to make a decision based on the absolute scale of things, then using magicalfactorx (guessing on the meaning of this abbreviation) is probably good.
If you want to make a decision based on the logical 'zoom' level, consider `scalingFactor`.

Reply:
Thanks, this is really helpful actually. Yea, you're guess is right, but I think the `scalingFactor` would allow for better one to one mapping with what design wants when they create their mocks